tools/update-lib-configs.js

```diff
@@ -88,6 +88,10 @@ categories.forEach((category) => {
 })
 
 // Format files.
-const linter = new eslint.CLIEngine({ fix: true })
-const report = linter.executeOnFiles([ROOT])
-eslint.CLIEngine.outputFixes(report)
+async function main() {
```

Comment:
This process is not the "main" of this script, so can you rename it to "format" etc.?

Reply:
done